cypress-io/cypress
# feat: (launchpad) add modal to display testing type differences

- Closes https://cypress-io.atlassian.net/browse/UNIFY-445

### Additional details

This PR:

- Fixes the color purple, values 50-1000 were reversed
- Adds some icons
- Adds a new StandardModal component (Jess contributed to this also)
- Adds components for Compare Testing Types modal, and a tiny "inline code fragment" for `doing this`.
- Cleans up the main Testing Type cards a little and allows them to trigger the Testing Type Compare modal with the new question mark icon
- Adds images to the testing type cards

### How to test

- Go to the welcome page for a project in the launchpad
- Click "Review the differences between each testing type"
- Modal should appear showing both testing types and example
- Click outside, or the X or press space to clear the modal
- Use one of the question mark buttons in the Testing Type Cards to open the modal also

### How has the user experience changed?

Demo:

https://user-images.githubusercontent.com/8340719/137384111-8a9cb654-b347-4096-907e-b69aca589c76.mov

### PR Tasks
